src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
```

Comment:
You use the main class to test your solution. But don’t push it to GitHub! Let’s remove this class